cookbooks/cdo-apps/templates/default/unicorn.sh.erb

```diff
@@ -40,6 +40,12 @@ run () {
 
 case $action in
 start)
+<% if node['cdo-apps']['nginx_enabled'] %>
+        if [ ! -d "/run/unicorn" ]
+        then
+            mkdir -p /run/unicorn; chown ${AS_USER}: /run/unicorn
```

Comment:
Does this also set the group to ubuntu?  (Not sure if that is essential anyway.)